verify/builtin/verify_all.rb

```diff
@@ -0,0 +1,244 @@
+require 'optparse'
```

Comment:
I am really confused why you are writing these kind of control scripts in Ruby. It is adding unnecessary dependencies.